run_cperf

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python
+# ===--- run --------------------------------------------------------------===
```

Comment:
run -> run_cperf

Reply:
fixed in my workspace! I'll be submitting a few more iterations of this while I'm getting it up and running, will include that in next commit. Thanks!